gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dform_ttt2.lua

```diff
@@ -474,6 +474,43 @@ function PANEL:MakeColorMixer(data)
 	return right, left
 end
 
+---
+-- Will be removed after #815 is merged
```

Comment:
I'll take care of this either in my PR if this is merged first or in this PR if mine is merged first